Alignment/OfflineValidation/test/testZmumu.yaml

```diff
@@ -0,0 +1,61 @@
+LFS: /eos/cms/store/group/alca_trackeralign/AlignmentValidation/AlignmentValidation/NewTool
+name: TestZmumu
+alignments:
+    PromptReco:
+        color: 600
+        globaltag: 111X_dataRun3_Prompt_v5
+        style: 2101
+        title: Alignment during data taking
+    ReReco:
+        color: 632
+        globaltag: 113X_dataRun2_v1
+        style: 2101
+        title: End-of-year re-reconstruction
+    ULRun2:
+        color: 418
+        conditions:
+            TrackerAlignmentRcd:
+                connect: frontier://FrontierProd/CMS_CONDITIONS
+                label: 
+                tag: TrackerAlignment_v28_offline
+            TrackerAlignmentErrorExtendedRcd:
+                connect: frontier://FrontierProd/CMS_CONDITIONS
+                label: 
+                tag: TrackerAlignmentExtendedErrors_v15_offline_IOVs
+        globaltag: 113X_dataRun2_v1
+        style: 2101
+        title: Legacy reprocessing
+
+validations:
+    Zmumu:
+        merge:
+            Legacy:
+                methods:
```

Comment:
Where is this option used in your code? I haven't found it.

Check this for all options: it would make no sense to give options that aren't used...